pylint/checkers/refactoring.py

```diff
@@ -297,6 +298,12 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "following a chain of ifs, all of them containing a "
             "continue statement.",
         ),
+        "R1725": (
+            "Consider using generator %s(%s)",
+            "consider-using-generator",
+            "Comprehension inside of any or all calls is unnecessary "
+            "and should be replaced by a generator",
```

Comment:
Most of the other message descriptions give a reason why the warnings exists. Should we do the same here?
We could mention that the a performance benefit from not needing to create a whole list, only for it to be used once and thrown away.

Reply:
What about `and should be replaced by a generator` => `a generator would be sufficient and faster.`